src/algorithms/crypto/vrf.id

```diff
@@ -13,6 +13,8 @@ type VRFKeyPair struct {
 }
 
 type VRFResult struct {
+    Output            Bytes  // @(internal)
```

Comment:
This was removed in the linked PR, however `Output` is still being referred to from other places, so I am reverting it back.